code/game/machinery/doors/airlock.dm

```diff
@@ -1022,7 +1022,7 @@ There are 9 wires.
 				playsound(loc, 'sound/items/keychainrattle.ogg', 30, 1, -2)
 			else
 				playsound(loc, 'sound/items/keychainrattle.ogg', 700, 1, -2)
-			if(do_after(user, 300, src)) //in ms so half a min of sitting their trying
+			if(do_after(user, 250, src)) //in ms so half a min of sitting their trying
```

Comment:
`>:T`

Reply:
use words

Reply:
me dislike `>:T`

This should not be nerfed as its easy to spam

Reply:
but it's a buff. Unless I am misunderstanding how do_afters work

Reply:
She means nerfed as in 'the value shouldnt be reduced'. I disagree, personally-  many people have complained about the keys being useless even to janitors

Reply:
I agree. My first step of the day as janitor is chucking those keys. It's not worth carrying them around. I'd say they are STILL not worth carrying but they are least not totally infuriating to use with this. By the time you find the key you could have just smashed the door, hacked it (not hard to afford gloves as a janitor) or just asked someone to come open it. This still gives plenty of time for the jingles to attracted someone.

Reply:
Also as far as I'm aware these keys are mostly balanced around the Eris. We have a lot more doors and much bigger departments and the janitors not trying to clean one single mess they need into every single room to call it clean. It becomes a nightmare trying to use them for this.

Reply:
"Easy to spam"? How can "clicking once, tabbing out, cooking dinner, filing taxes, taking a nap, forgetting you're playing, clicking again, ..." be considered spamming? Yes, the keys are uselessly ... useless. A ridiculous solution to a non-problem. Janitor keeps stealing your stuff? Get them arrested and job-banned.
And don't get me started about that fucking keyring that's as big as the fucking vacuum.
Or the use of secure doors seemingly only "because they are blue'.
So in short, I agree with benj8560 and would buff it even more, thank you.